projects/angular/src/data/datagrid/datagrid.html

```diff
@@ -76,21 +76,25 @@
           </div>
 
           <div role="presentation" class="datagrid-rows">
-            <clr-dg-row class="datagrid-loading-more-items-row" *ngIf="loadingMoreItems">
-              <clr-dg-cell class="datagrid-loading-more-items-cell">
-                <clr-spinner [clrMedium]="true"></clr-spinner>
-                <span>{{ commonStrings.keys.loading }}</span>
-              </clr-dg-cell>
-            </clr-dg-row>
+            <div role="row" class="datagrid-row datagrid-row-loading" *ngIf="loadingMoreItems">
+              <div class="datagrid-row-master datagrid-row-flex">
+                <div role="gridcell" class="datagrid-cell">
+                  <clr-spinner [clrMedium]="true"></clr-spinner>
+                  <span>{{ commonStrings.keys.loading }}</span>
+                </div>
+              </div>
+            </div>
 
             <ng-container #displayedRows></ng-container>
 
-            <clr-dg-row class="datagrid-loading-more-items-row" *ngIf="loadingMoreItems">
-              <clr-dg-cell class="datagrid-loading-more-items-cell">
-                <clr-spinner [clrMedium]="true"></clr-spinner>
-                <span>{{ commonStrings.keys.loading }}</span>
-              </clr-dg-cell>
-            </clr-dg-row>
+            <div role="row" class="datagrid-row datagrid-row-loading" *ngIf="loadingMoreItems">
+              <div class="datagrid-row-master datagrid-row-flex">
+                <div role="gridcell" class="datagrid-cell">
+                  <clr-spinner [clrMedium]="true"></clr-spinner>
```

Comment:
it doesn't seem like part of this PR's change but maybe it could be simplified to just `clrMedium` instead of `[clrMedium]="true"` like on line 108

Reply:
Thanks, Resolved in aaf2d89